### ISSUE TITLE
Reproducing the evaluation result

### ISSUE THREAD

**Issue by hyukkyukang** [beerqa/IRRR#7]
Hello, I've followed the instruction in the README and got below result for the evaluation:

{'em': 0.013030238651653566, 'f1': 0.04761130103682215, 'prec': 0.05880147367704435, 'recall': 0.05300143765695587, 'hotpotqa_em': 0.02821840040073468, 'hotpotqa_f1': 0.07495189473787742, 'hotpotqa_prec': 0.08299098878209452, 'hotpotqa_recall': 0.08642775380127182, 'squad_em': 0.0018445646827348746, 'squad_f1': 0.02747568671370147, 'squad_prec': 0.04098654426679513, 'squad_recall': 0.02838385189843272, 'macro_em': 0.015031482541734777, 'macro_f1': 0.051213790725789445, 'macro_prec': 0.06198876652444482, 'macro_recall': 0.05740580284985227}

Has anyone faced similar problem?

**Comment by hyukkyukang** [comment]
Hi, I've went through the inference process step-by-step and couldn't find anything wrong with the code.
However, I just figured out that the output logits from the trained model is just bad. 
It generates high probability for incorrect answers.

Can you please tell me what accuracy I am supposed to get when using the checkpoint provided in the repository?
@qipeng

**Comment by hyukkyukang** [comment]
Hello,

The above problem happened because I've changed layer normalization code when I was trying to remove deprecated warnings from the tensorflow library.

Original code:
```python
return tf.contrib.layers.layer_norm(
          inputs=input_tensor, begin_norm_axis=-1, begin_params_axis=-1, scope=name)
```

My modification:
```python
return tf.keras.layers.LayerNormalization(axis=-1, name=name)(input_tensor)
```

Please refer to the below part of the code:
https://github.com/beerqa/IRRR/blob/76b288e5e2508ab62a52b7d6dc7d69065c0543e2/model/modeling.py#L403


After changing back to the original code, I was able to reproduce the accuracy reported in the Table 5 of the author's paper.
```JSON
{
    "em": 0.5226258763543659,
    "f1": 0.6269750350973999,
    "prec": 0.6428464236940972,
    "recall": 0.6405468465933917,
    
    "hotpotqa_em": 0.5695441643012189,
    "hotpotqa_f1": 0.6810414745462648,
    "hotpotqa_prec": 0.7025768822187252,
    "hotpotqa_recall": 0.6888242701066212,
    
    "squad_em": 0.4880718150516478,
    "squad_f1": 0.5871565518387571,
    "squad_prec": 0.5988565422253318,
    "squad_recall": 0.604991818381301,
    
    "macro_em": 0.5288079896764334,
    "macro_f1": 0.634099013192511,
    "macro_prec": 0.6507167122220285,
    "macro_recall": 0.6469080442439611
}
```
